modules/oxide/src/less/skins/content/dark/content.less

```diff
@@ -85,3 +85,11 @@ code {
     padding-right: 1rem;
   }
 }
+
+li {
+  color: #ffffff;
```

Comment:
Hey @ArvinJ-H  I _think_ we could potentially reduce some of the code duplication here by using mixins and imports.

Could we define a `li` style with everything from `background-color` to `font-family` and then just update the `color` in each file as needed while just referencing the mixin for all of the remaining styles that we need? I'm not sure if it's a convention we follow here but could help with that.